.github/workflows/nightly.yml

```diff
@@ -343,8 +401,39 @@ jobs:
           # Hack to disable a test that hangs
           php -r '$c = file_get_contents("tests/Filesystem/FilesystemTest.php"); $c = str_replace("*/\n    public function testSharedGet()", "* @group skip\n     */\n    public function testSharedGet()", $c); file_put_contents("tests/Filesystem/FilesystemTest.php", $c);'
           export ASAN_OPTIONS=exitcode=139
-          php vendor/bin/phpunit --exclude-group skip || EXIT_CODE=$?
-          if [ $EXIT_CODE -gt 128 ]; then
+          php $GITHUB_WORKSPACE/.github/patch.php vendor/bin/phpunit --exclude-group skip || EXIT_CODE=$?
```

Comment:
I understand you're targetting 8.1 because patch.php is needed on each branch. But the rest of the changes in nightly.yml seem unnecessary, as nightly.yml from older branches is essentially never used.

Reply:
I thought it would be ok, since it would get merged into master anyway...